source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj

```diff
@@ -21,10 +21,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Common" Version="0.33.0" PrivateAssets="All" />
-    <PackageReference Include="Cake.Core" Version="0.33.0" PrivateAssets="All" />
-    <PackageReference Include="NuGet.Packaging" Version="5.0.2" PrivateAssets="All" />
-    <PackageReference Include="NuGet.Protocol" Version="5.0.2" PrivateAssets="All" />
+    <PackageReference Include="Cake.Common" Version="0.38.5" PrivateAssets="All" />
+    <PackageReference Include="Cake.Core" Version="0.38.5" PrivateAssets="All" />
```

Comment:
Can we target Cake 0.36.0 instead? That [would still be compatible with NuGet 5.4.0](https://github.com/cake-build/cake/blob/v0.36.0/src/Cake.NuGet/Cake.NuGet.csproj) and would keep it compatible with from Cake 0.36.0 to 0.38.5?

Reply:
that's a great idea! I'll update the PR. (If I find out how)

Reply:
@MCCoolMoDAue  if you add another commit to your fork and push that, the PR will update automatically.